### DISCUSSION TITLE
Using querydsl with OpenMdx JDO

### DISCUSSION THREAD

**Discussion by fschon** [openmdx/openmdx#75] [Ideas]
I see that querydsl has a binding for JDO and one can create very easy queries for JDO objects. See: http://querydsl.com/static/querydsl/2.3.0/reference/html/ch02s02.html

I tried to run the querydsl JDOAnnotationFinder against opencrx and there were no JDO objects found. I guess the objects need to be annotated with @PersistenceCapable.

Not knowing how the openmdx JDO implementation works and how standard it is, would it be possible to simply annotate the JDO objects with @PersistenceCapable during code generation phase and then get querydsl queries out the box?

Thanks for clarifying.

**Comment by DlRTY-HARRY** [comment]
openMDX has its own typed query language and is incompatible with Qerydsl.
